### PR TITLE
Add Gradle instructions

### PR DESCRIPTION
**Pull request by cies**
I saw there was a Gradle setup but no instructions for using Gradle. We use Gradle so I took the time to (hopefully) help you out by adding instructions for creating an MCVE with Gradle.

### CODE REVIEW

**Comment by lukaseder** [comment]
Thanks for your suggestions. I'm sure this will be useful, I'll check it out soon.

**Comment by cies** [comment]
No rush, git it patient. I just wanted to make it easier for *both* myself and others.

I hope it helps you, as jOOQ also helps us. :)

**Comment by cies** [comment]
Small thing that I thought of that can make it easier to produce MCVEs with custom generators: give each subproject a custom generator that simply inherits the defaults, and add an optional step to the README "copy over your custom generator".

I have no idea though how often MCVEs are posted that have a custom generator.

**Comment by lukaseder** [comment]
> I have no idea though how often MCVEs are posted that have a custom generator.

It is hardly ever needed, and even if people start writing their own, there's usually a better way (including feature requests). The MCVE can already be overwhelming enough with DB setup, testcontainers, etc. etc. I prefer not to add any more complexity.

**Comment by lukaseder** [comment]
I've had a look at it. I don't like how things are repeated now, it'll be harder to maintain when stuff is duplicated.

Most MCVEs aren't about Maven or Gradle, but about jOOQ, so these two build systems shouldn't take so much space in the descriptions.

I'll take the relevant bits from your suggestion and merge them in the Maven specific text.

File: README.md [hunk 1]
```diff
@@ -1,42 +1,61 @@
...
+
+- Uses testcontainers (for MySQL, Oracle, PostgreSQL, SQL Server) to set up a database instance, and makes it available to jOOQ and JUnit.
+- Installs a sample schema located in `src/main/resources/db/migration` into an file-based H2 database or testcontainers managed MySQL, Oracle, PostgreSQL, SQL Server database.
+- Runs jOOQ's code generator on it.
+- Runs a simple integration test.
+
+This should work without any additional setup on your side, other than having a JDK, build tool (only when using Maven), and Docker available.
+If you need to have the commercial distribution of jOOQ installed, e.g. for using Oracle or SQL Server, that must be installed as well.
+In case you do not have Docker, you can still run the H2 based examples directly, ignoring the MySQL, Oracle, PostgreSQL, SQL Server based ones.
+
```
Comment (lukaseder) [inline README.md:18]: I understand why you wrote this, and for some people who like to read Macbeth in its original text, this is nice. But for most people, I'm positive it's just noise. They know why they're doing this, so they just want to get the minimal information they need.

File: README.md [hunk 2]
```diff
@@ -1,42 +1,61 @@
...
 
 ## How to use this project to prepare your MCVE
 
-Create a fork from this project and then
+
+First you need to create a fork of this project (do this from the [GitHub web UI](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo))
+so you can publish your MCVE once done. Then clone the repository you just forked and `cd` into it:
+
```
Comment (lukaseder) [inline README.md:25]: Here, too, I understand why someone might want to do this. But "forking" is a very well known concept on github, and I just don't want to create noise about well-known concepts. We don't have to explain git either, or pull, or `cd`, I think?

Besides, such a link is quickly outdated, and I'll just have to maintain it going forward.

File: README.md (hunk 1 above)
Comment (lukaseder) [reply README.md:18]: I just realise, I wrote this myself, further down 😅 Jeez, I wonder if we really need it.

File: README.md [hunk 3]
```diff
@@ -63,6 +82,68 @@ When you've set up your MCVE, run these statements again:
...
+For your MCVE, you will have to adapt a few things, probably. This may include:
+
+- The Java version (only if the Java version is important for your MCVE):
+  - Go to the relevant subproject's build file (i.e. `$REPO_ROOT/<relevant subproject>/build.gradle.kts`) and add the following line (adapting the version numer) somewhere after the `import` statements:
+
+
+```kotlin
+java.toolchain.languageVersion = JavaLanguageVersion.of(17)
+```
```
Comment (lukaseder) [inline README.md:107]: There are a ton of ways to do this, and they're language specific. This stuff shouldn't be described with words, but by example.

It's much better to have a default that already works, and then show the user where they can adapt stuff. I'll do this the day it really matters. Until then, let's just work with the defaults.

**Pull request closed without merging**